src/cbrkit/sim/collections.py

```diff
@@ -247,7 +364,7 @@ def __call__(self, query: Sequence[V], case: Sequence[V]) -> float:
 
 
 @dataclass(slots=True, frozen=True)
-class SequenceSim[S: Float](StructuredValue[float]):
+class SequenceLocalSim[S: Float](StructuredValue[float]):
```

Comment:
Do we still need this class? From my understanding of the code, the following should be equivalent because providing the mapping is optional:

```python
SequenceSim(1, [0, 1, 1]) == SequenceLocalSim(1, [0, 1, 1])
```

If so, we should probably just use `SequenceSim` and drop `SequenceLocalSim`.

Reply:
I wanted to keep it to avoid changing other classes, but i could change them in a way that they can handle this change